### PR TITLE
Fixed documentation for batch_normalization

### PR DESCRIPTION
**Pull request by DreSimpelo**
Fixed batch_normalization part of the nn python Documentation.

Fixed syntax errors restricting the latex code to load in the markdown documentation file for "tf.nn.batch_normalization(...)".

### CODE REVIEW

File: tensorflow/g3doc/api_docs/python/nn.md [hunk 1]
```diff
@@ -3221,9 +3221,9 @@ Batch normalization.
 
 As described in http://arxiv.org/abs/1502.03167.
 Normalizes a tensor by `mean` and `variance`, and applies (optionally) a
-`scale` \\\\(\gamma\\\\) to it, as well as an `offset` \\\\(\\beta\\\\):
+`scale` \\(\gamma\\) to it, as well as an `offset` \\(\beta\\):
 
-\\\\(\\frac{\gamma(x-\mu)}{\sigma}+\\beta\\\\)
+$$\frac{\gamma(x-\mu)}{\sigma}+\beta$$
```
Comment (jhseu) [inline tensorflow/g3doc/api_docs/python/nn.md]: Can you leave it as \\( and \\) instead of switching to $$?

**Review by jhseu — CHANGES_REQUESTED** [review]
Thanks for the change! The API docs are autogenerated. You'll have to change this file instead:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/nn.py#L939

File: tensorflow/g3doc/api_docs/python/nn.md (hunk 1 above)
Comment (DreSimpelo) [inline tensorflow/g3doc/api_docs/python/nn.md]: Done! (The equation is no longer centered after that change)

**Comment by DreSimpelo** [comment]
Moved: https://github.com/tensorflow/tensorflow/pull/4689

**Pull request closed without merging**